source/draw/gpu/opengl/OpenGLSurface.ooc

```diff
@@ -50,14 +50,18 @@ OpenGLSurface: abstract class extends GpuCanvas {
 		this draw(destination, map)
 	}
 	drawLines: override func (pointList: VectorList<FloatPoint2D>) {
-		f := func { this context drawLines(pointList, this _projection * this _toLocal, this pen) }
-		this draw(f)
-		(f as Closure) free()
+		this _bind()
+		this context backend setViewport(this viewport)
+		this context backend enableBlend(false)
+		this context drawLines(pointList, this _projection * this _toLocal, this pen)
+		this _unbind()
```

Comment:
Inlined the old draw call to get rid of the `draw` dependency. Getting rid of `_projection` will require breaking the API in another PR after making and adapting to a new line drawing method.